packages/search-input/src/SearchInputClearButton.tsx

```diff
@@ -21,17 +21,20 @@ export function SearchInputClearButton({
 			role="button"
 			alignItems="center"
 			justifyContent="center"
-			aria-label="Clear input"
+			aria-label="Clear search"
```

Comment:
Updating this while I'm here to be more consistent with https://react-spectrum.adobe.com/react-spectrum/SearchField.html